docs/proposals/20230612-node-os-image-caching.md

```diff
@@ -86,6 +86,7 @@ We will know we’ve succeeded when we can benchmark speed increases and success
 1. Optimization for efficiency and scalability
 1. A more complicated method of selecting a good candidate node such as incorporating manual forced prototype creation, which would perform better but take research to find the optimal method
 1. Optimization of the default time interval into a specific, best general default which would have to be researched
+1. Bad snapshot rollbacks since it will be hard to generically determine when a snapshot is bad currently
```

Comment:
It may still be worth trying to come up with a manual rollback method, but I think it's the "automatic bad snapshot detection" piece that we should definitely call out of scope:
```suggestion
1. Automatic bad snapshot rollbacks since it will be hard to generically determine when a snapshot is bad currently
```